roles/wildfly_systemd/defaults/main.yml

```diff
@@ -40,3 +40,9 @@ wildfly_instance_name: wildfly
 wildfly_node_id: "{{ inventory_hostname_short }}"
 wildfly_service_name: "{{ wildfly_instance_name }}"
 wildfly_basedir_prefix: "{{ wildfly_home }}"
+
+wildfly_systemd_port: 9990
```

Comment:
Please rename because it can be misleading as it is. I'd prefer wildfly_systemd_wait_port or something similar that indicates it's the port waited for